pom.xml

```diff
@@ -391,6 +394,36 @@
         <scope>provided</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
+        <groupId>io.opencensus</groupId>
```

Comment:
Can we use bom to pin all opencensus dependency versions (see how we do this for grpc deps)?

Reply:
I don't think bom is supported from opencensus [open issue](https://github.com/census-instrumentation/opencensus-java/issues/1117). I couldn't find it on mvn central either